client/src/features/ProjectPageV2/ProjectPageContent/ProjectInformation/ProjectInformation.tsx

```diff
@@ -42,6 +46,70 @@ export function ProjectImageView() {
   );
 }
 
+function ProjectInformationMember({
+  member,
+}: {
+  member: ProjectMemberResponse;
+}) {
+  const classNames = ["fw-bold", "mb-1"];
+  if (member.last_name != null) {
+    return (
+      <div className={cx(classNames)}>
+        {member.first_name ?? ""} {member.last_name}
+      </div>
+    );
+  }
+  if (member.email != null) {
+    return <div className={cx(classNames)}>{member.email}</div>;
+  }
+  return <div className={cx(classNames)}>{member.id}</div>;
```

Comment:
why not compute a `displayName` and use that for rendering to avoid repetition?

```suggestion
  const displayName =
    member.first_name && member.last_name
      ? `${member.first_name} ${member.last_name}`
      : member.last_name
      ? member.last_name
      : member.email
      ? member.email
      : member.id;

  return <div className={cx("fw-bold", "mb-1")}>{displayName}</div>;
```